### PR TITLE
AN-628: add helper text when adding new database

### PR DESCRIPTION
**Pull request by lpoulain**
Adds helper text for the catalogs and schemas when adding a new database of type Starburst

### CODE REVIEW

File: drivers/starburst/resources/metabase-plugin.yaml [hunk 1]
```diff
@@ -21,9 +21,11 @@ driver:
           placeholder: tpch
           display-name: Catalog
           required: true
+          helper-text: Starburst Catalogs contain schemas and reference data sources via a connector.
     - name: schema
       display-name: Schema (optional)
       required: false
+      helper-text: Starburst Schemas contain tables
```
Comment (andrewdibiasio6) [inline drivers/starburst/resources/metabase-plugin.yaml:28]: Can we use the `Only add tables to Metabase that come from a specific schema.`

**Comment by andrewdibiasio6** [comment]
LGTM pending the small change.

**Review by andrewdibiasio6 — APPROVED** [review]

File: drivers/starburst/resources/metabase-plugin.yaml (hunk 1 above)
Comment (lpoulain) [reply drivers/starburst/resources/metabase-plugin.yaml]: Done

File: drivers/starburst/resources/metabase-plugin.yaml [hunk 2]
```diff
@@ -21,9 +21,11 @@ driver:
           placeholder: tpch
           display-name: Catalog
           required: true
+          helper-text: Starburst Catalogs contain schemas and reference data sources via a connector.
     - name: schema
       display-name: Schema (optional)
       required: false
+      helper-text: Only add tables to Metabase that come from a specific schema
```
Comment (hovaesco) [inline drivers/starburst/resources/metabase-plugin.yaml:28]: nit: please be consistent with dot/no dot at the end

File: drivers/starburst/resources/metabase-plugin.yaml [hunk 3]
```diff
@@ -21,9 +21,11 @@ driver:
           placeholder: tpch
           display-name: Catalog
           required: true
+          helper-text: Starburst Catalogs contain schemas and reference data sources via a connector.
```
[the remark this answers is not in the document]
Comment (lpoulain) [reply drivers/starburst/resources/metabase-plugin.yaml:24]: And yet we call it the Starburst connector. @andrewdibiasio6 @aalbu , thoughts?

File: drivers/starburst/resources/metabase-plugin.yaml (hunk 2 above)
Comment (lpoulain) [reply drivers/starburst/resources/metabase-plugin.yaml:28]: Any suggestion of how it should be worded then?

Comment (lpoulain) [reply drivers/starburst/resources/metabase-plugin.yaml]: Done

**Review by hovaesco — APPROVED** [review]

**Pull request merged**